contracts/mocks/MockBlockRelay.sol

```diff
@@ -228,4 +241,28 @@ contract MockBlockRelay is BlockRelayInterface {
     }
     return _root == tree;
   }
+
+  /// @dev This function checks if the relayer has been paid
+  /// @param _blockHash Hash of the block header
+  /// @return true if the relayer has been paid, false otherwise
```

Comment:
```suggestion
  /// @dev This function checks if the relayer has been paid.
  /// @param _blockHash Hash of the block header.
  /// @return true if the relayer has been paid, false otherwise.
```